modules/multi-finality-verifier/src/lib.rs

```diff
@@ -202,8 +206,20 @@ pub mod pallet {
 				gateway_id
 			);
 
-			Ok(().into())
-		}
+            let now: u64 = T::TimeProvider::now().as_secs();
```

Comment:
Use block number instead of seconds - problematic to come to consensus on that value otherwise